consensus/merge/build.gradle

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+plugins { id 'java' }
+
+jar {
+  archiveBaseName = 'besu-merge'
+  manifest {
+    attributes(
+      'Specification-Title': archiveBaseName,
+      'Specification-Version': project.version,
+      'Implementation-Title': archiveBaseName,
+      'Implementation-Version': calculateVersion()
+      )
+  }
+}
+
+repositories { mavenCentral() }
+
+dependencies {
+  implementation project(':config')
```

Comment:
just a question that comes to mind. are we sure we need all these dependencies

Reply:
Trimmed the dependencies to minimal :+1:  We can add stuff back as it becomes necessary